src/driver/franka_plan_runner.h

```diff
@@ -263,6 +263,9 @@ class FrankaPlanRunner {
   const std::array<double, 6> kMediumForceThreshold {40.0, 40.0, 40.0,
                                                      50.0, 50.0, 50.0};
 
+  const std::array<double, 7> kJointTorqueLimits {87.0, 87.0, 87.0, 87.0,
+                                                  12.0, 12.0, 12.0};
+
```

Comment:
Why need this? Already have 
```c++
  // Collision torque thresholds for each joint in [Nm].
  const std::array<double, 7> kHighTorqueThreshold {100.0, 100.0, 100.0, 100.0,
                                                    100.0, 100.0, 100.0};
  const std::array<double, 7> kMediumTorqueThreshold {40.0, 40.0, 36.0, 36.0,
                                                      32.0, 28.0, 24.0};
```
defined above

Reply:
But these are the actual franka torque limits
https://frankaemika.github.io/docs/control_parameters.html#constants